pandas/core/dtypes/common.py

```diff
@@ -1685,6 +1685,38 @@ def is_extension_type(arr):
     return False
 
 
+def is_extension_array_dtype(arr_or_dtype):
+    """Check if an object is a pandas extension array type
+
+    Parameters
+    ----------
+    arr_or_dtype : object
+
+    Returns
+    -------
+    bool
+
+    Notes
+    -----
+    This checks whether an object implements the pandas extension
+    array interface. In pandas, this includes:
+
+    * Categorical
+    * PeriodArray
+    * IntervalArray
+    * SparseArray
+
+    Third-party libraries may implement arrays or types satisfying
+    this interface as well.
+    """
+    from pandas.core.arrays import ExtensionArray
+
+    # we want to unpack series, anything else?
+    if isinstance(arr_or_dtype, ABCSeries):
+        arr_or_dtype = arr_or_dtype.values
```

Comment:
This is `Series.values`, what else would it return? An object-typed NumPy array? I think the ship has sailed on `Series.values` always being a NumPy array.

Reply:
Let's use `Series._values` for now? That gets the values of the block, and is certainly an ExtensionArray in case the series holds one. 
The we can postpone the decision on what `.values` returns?